app/components/windows/Settings.vue.ts

```diff
@@ -79,6 +79,8 @@ export default class SceneTransitions extends Vue {
   @Watch('categoryName')
   onCategoryNameChangedHandler(categoryName: string) {
     this.settingsData = this.settingsService.getSettingsFormData(categoryName);
+    const el = this.$refs.settingsContainer as HTMLElement;
```

Comment:
define `refs` as a component property
```
$refs: {
 settingsContainer: HTMLElement;
}
```

Reply:
i'm not sure where i would put that snippet in the code

Reply:
https://github.com/vuejs/vue-class-component/issues/94